### ISSUE TITLE
Java crash when running PCUAE V4 install from PCUAE manager (2.1.7.0) on W11

### ISSUE THREAD

**Issue by theshinyknight** [lantzelot-swe/PCUGameManager#22]
Getting this error when using the PCUAE menu to install it to a USB thumbdrive. The same happens if I start PCUAE manager and clicking YES on the 4.0 install prompt.

This is a fresh install of PCUAE manager

```
Something went wrong during install.

java.util.concurrent.ExecutionException: java.io.IOException: Cannot run program ".\pcuae-install\A-How-to-Use-Flash-PCUAE-nandb-on-THE400-Mini.txt" (in directory ".\pcuae-install"): CreateProcess error=193, %1 is not a valid Win32 application
	at dyorgio.runtime.out.process.OneRunOutProcess.call(OneRunOutProcess.java:277)
	at dyorgio.runtime.run.as.root.RootExecutor.execute(RootExecutor.java:148)
	at dyorgio.runtime.run.as.root.RootExecutor.call(RootExecutor.java:133)
	at se.lantz.manager.pcuae.BaseInstallManager.runAndWaitForInstallation(BaseInstallManager.java:317)
	at se.lantz.manager.pcuae.BaseInstallManager.lambda$downloadLatestVersion$5(BaseInstallManager.java:490)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
Caused by: java.io.IOException: Cannot run program ".\pcuae-install\A-How-to-Use-Flash-PCUAE-nandb-on-THE400-Mini.txt" (in directory ".\pcuae-install"): CreateProcess error=193, %1 is not a valid Win32 application
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
	at java.base/java.lang.Runtime.exec(Runtime.java:590)
	at java.base/java.lang.Runtime.exec(Runtime.java:414)
	at se.lantz.manager.pcuae.InstallCallable.call(InstallCallable.java:25)
	at se.lantz.manager.pcuae.InstallCallable.call(InstallCallable.java:8)
	at dyorgio.runtime.run.as.root.RootExecutor$RemoteCall.call(RootExecutor.java:170)
	at dyorgio.runtime.run.as.root.RootExecutor$RemoteCall.call(RootExecutor.java:155)
	at dyorgio.runtime.out.process.entrypoint.OneRunRemoteMain.main(OneRunRemoteMain.java:58)
Caused by: java.io.IOException: CreateProcess error=193, %1 is not a valid Win32 application
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:483)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:158)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1107)
	... 8 more

```

**Comment by lantzelot-swe** [comment]
Ok, it picks the wrong file from the PCUAE github project. I have corrected it in 2.17.1.

**Comment by theshinyknight** [comment]
Thanks
